server/data/Users.txt

```diff
@@ -3,5 +3,4 @@
 948bf810-71e2-11ea-a8e0-73473bc162b1,null,false,false,dev,dev,dev@gmail.com,13901,player_icon_1,0,0,2020-04-16T00:27:57.206Z,2020-04-16T00:19:28.496Z,2020-03-29T17:27:35.698Z,false,
 9e9a98d0-71e6-11ea-b474-933b66303ca9,null,false,false,john,john,john@gmail.com,21663,player_icon_11,0,0,2020-04-16T01:01:34.389Z,2020-04-06T18:28:32.603Z,2020-03-29T17:56:30.557Z,true,
 8fd55dc0-71f6-11ea-a4ad-f1607ee8ca58,null,false,false,mack,mack,mack@gmail.com,78760,player_icon_7,0,0,2020-04-16T00:22:24.412Z,2020-04-16T00:19:57.298Z,2020-03-29T19:50:37.725Z,false,
-9de8ed60-7399-11ea-ac9d-816463fcbcf7,null,false,true,user,pass,user@gmail.com,100,player_icon_2,0,0,2020-04-07T19:48:34.264Z,2020-04-07T19:48:34.263Z,2020-03-31T21:50:20.470Z,false,
-
+9de8ed60-7399-11ea-ac9d-816463fcbcf7,null,false,true,user,pass,user@gmail.com,100,player_icon_2,0,0,2020-04-07T19:48:34.264Z,2020-04-07T19:48:34.263Z,2020-03-31T21:50:20.470Z,false,
```

Comment:
To not reintroduce the bug you've fixed, is there empty space at the bottom of this file?

Reply:
not that i can see